lib/iris/tests/integration/test_trajectory.py

```diff
@@ -216,8 +216,10 @@ def setUp(self):
         cube.coord("depth").bounds = b32
         self.cube = cube
         # define a latitude trajectory (put coords in a different order
-        # to the cube, just to be awkward)
-        latitudes = list(range(-90, 90, 2))
+        # to the cube, just to be awkward) although avoid south pole
+        # singularity as a sample point and the issue of snapping to
+        # multi-equidistant closest points from within orca antarctic hole
+        latitudes = list(range(-80, 90, 2))
```

Comment:
This test is an accident waiting to happen... let's just make it a wee bit more robust and avoid a known possible problem.